pages/index.tsx

```diff
@@ -82,20 +79,18 @@ const Entry = ({ doc }) => {
         <h3>{doc.title}</h3>
       </Link>
       <p>Posted on {new Date(doc.pubDate).toLocaleDateString()}</p>
-      <p
-        dangerouslySetInnerHTML={{
-          __html: doc.content.replace(/^"/, '').replace(/"$/, '')
-        }}
-      ></p>
+      <p>
+        {excerpt(doc.contentSnippet?.replace(/^"/, '').replace(/"$/, ''), 50)}
```

Comment:
This could be shorter. Something like:

```
{excerpt(doc.contentSnippet?.replace(/^"|"$/, ''), 50)}
```

But then. why are we removing those things again? :thinking:

Reply:
Oh that a good one, apparently the reason we need to remove those is, some RSS feed hash `contentSnippet` as a literal `""` empty quotes (my blog :joy:) so we need to remove it.

Reply:
Apparently I came up with a better approach  to solve this problem, it will be a new method to enforce the minimum length for the excerpt, anything less than the minimum length will be treated as an empty string.

Reply:
:heavy_check_mark: